src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import Main from './components/main';
 import './i18n';
 
 function App() {
   return (
     <div>
       <h1>Online Books</h1>
+      <Main />
```

Comment:
you need to setup react router before you put the main page here 
and this should be separated task